src/quests/cognac/tasks/pld.ts

```diff
@@ -48,8 +49,7 @@ export class PLD {
         },
         post: () => {
           if (get("lastEncounter") === "The Furtivity of My City") {
-            this.gossip.incrementStench();
-            print(`Stench level increased to ${this.gossip.stench}.`);
+            print(`Stench level increased to ${get(CURRENT_STENCH)}.`);
```

Comment:
This isn't accurate; `CURRENT_STENCH` may not have been updated yet.

Reply:
Its true. Its a bit fuzzy as others are doing stuff at the same time. So it is never exactly correct, even in the old system I don't believe. I could update wording to 
print(`Stench level increased to approx. ${get(CURRENT_STENCH)}.`)

What do ya think? I like the idea of some local logging to help debug future issues, even if stench value isn't exactly correct

Reply:
I don't think its a big deal either way. If we wanted it to be even more accurate, could add it to the chatbot.

Reply:
I'll leave it as is for now. Realized session log will also have stench pref updates, which is data from chatbot side